meta-iot2000-bsp/kas.yml

```diff
@@ -0,0 +1,51 @@
+header:
+  version: 2
+
+machine:  iot2000
+
+repos:
+  meta-iot2000:
+    layers:
+      meta-iot2000-bsp:
+
+  poky:
+    url: https://git.yoctoproject.org/git/poky
+    refspec: 7d5822bf4cb2089687c3c9a908cf4a4ef4e9a53a
+    layers:
+      meta:
+      meta-poky:
+      meta-yocto-bsp:
+
+  meta-intel:
+    url: https://git.yoctoproject.org/git/meta-intel
+    refspec: 6add41510412ca196efb3e4f949d403a8b6f35d7
+
+  meta-openembedded:
+    url: http://git.openembedded.org/meta-openembedded
+    refspec: fe5c83312de11e80b85680ef237f8acb04b4b26e
+    layers:
+      meta-oe:
+
+bblayers_conf_header:
+  standard: |
+    POKY_BBLAYERS_CONF_VERSION = "2"
```

Comment:
Yes, for us, but these keys are configuration global valid. So IMO the cleaner approach is to specify in the key where those settings come from or what they define, this way overwriting this entry in files that include this file can be better specified without redundancy.